### ISSUE TITLE
Updating typo tolerance settings set minWordSizeForTypos to the default values in v0.27.0rc2

### ISSUE THREAD

**Issue by CaroFG** [meilisearch/meilisearch#2358]
**Describe the bug**
Updating typo tolerance settings (disableOnAttributes) set minWordSizeForTypos to the default values

**To Reproduce**
1. Update `minWordSizeForTypos` to override default
```
curl \
  -X POST 'http://localhost:7700/indexes/books_typo/settings/typo-tolerance' \
  -H 'Content-Type: application/json' \
  --data-binary '{
      "minWordSizeForTypos": {
          "oneTypo": 2,
          "twoTypos": 4
      }
  }'
```
2. Check  settings:
```
curl \
  -X GET 'http://localhost:7700/indexes/books_typo/settings/typo-tolerance'
{"enabled":true,"minWordSizeForTypos":{"oneTypo":2,"twoTypos":4},"disableOnWords":[],"disableOnAttributes":[]}%
```
3. Update `disableOnAttributes`
```
curl \
  -X POST 'http://localhost:7700/indexes/books_typo/settings/typo-tolerance' \
  -H 'Content-Type: application/json' \
  --data-binary '{
      "disableOnAttributes": ["isbn"]
  }'
```
4. Check settings
```
curl \
  -X GET 'http://localhost:7700/indexes/books_typo/settings/typo-tolerance'
{"enabled":true,"minWordSizeForTypos":{"oneTypo":5,"twoTypos":9},"disableOnWords":[],"disableOnAttributes":["isbn"]}%
```

**Expected behavior**
`minWordSizeForTypos` should have remained: `{"oneTypo":2,"twoTypos":4}`

**Meilisearch version:** v0.27.0rc2

**Comment by curquiza** [comment]
Thanks caro, a fix is on the way milli#520

**Comment by curquiza** [comment]
Reopening because not fixed yet on Meilisearch side

**Comment by curquiza** [comment]
Closed by #2366
